roles/ansible/tower/config-ansible-tower-ocp/README.md

```diff
@@ -46,6 +46,8 @@ The variables used to install Ansible Tower on OpenShift are outlined in the tab
 |pg_sslmode|SSL mode to be used in communication between Tower and PostgreSQL|no|prefer|
 |postgress_activate_wait|Time in seconds in which role will wait for PostgreSQL to become available during installation of Tower|no|120|
 |ansible_customization_file|Tower Installer may have some bugs in specific versions, this variable points to archive which holds an overlay if any Installer changes are needed|no|N/A|
+|kubernetes_awx_image|Container repository/image for custom Ansible Tower container images|no|Determined by Ansible Tower installer|
```

Comment:
to give the readers an example here, perhaps fill in the `defaults` column with something like: "example: registry.redhat.io/ansible-tower-38/ansible-tower-rhel7"

Reply:
Going to incorporate these suggestions into a new section of the README related to the items in the PR.